nss-day-cohort-60/rare-client-billowing-smurfs
# Author naming convention

### To test:

1. Launch app and log in
2. Navigate to User Management
3. Verify that users populate
4. Click on user and verify that correct info renders
